src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1469,20 +1460,16 @@ protected override void ProcessRecord()
                                     HttpResponseException httpEx = new(message, response);
                                     ErrorRecord er = new(httpEx, "WebCmdletWebResponseException", ErrorCategory.InvalidOperation, request);
                                     string detailMsg = string.Empty;
-                                    StreamReader reader = null;
                                     try
                                     {
-                                        reader = new StreamReader(StreamHelper.GetResponseStream(response));
-                                        // remove HTML tags making it easier to read
+                                        using StreamReader reader = new(StreamHelper.GetResponseStream(response));
```

Comment:
Now this will be compiled with two nested `try`? I'd prefer old code.

Reply:
reverted